New action Add Address to Group Cisco ASA (#627)
rapid7/insightconnect-plugins: cisco_asa/help.md

### Actions

#### Add Address to Group

> This action is used to add a preexisting Addresses to a Network Object Group from to allow them to block the host.

Reviewer: ```suggestion
This action is used to add a pre-existing IP address to a Network Object Group,
```

### Actions

#### Add Address to Group

This action is used to add an IP address associated with an existing Object to a Network Group.

> It works by checking the IPv4 and IPv6 addresses across all Objects in Cisco ASA. If a match is found, it adds the object to the group.

Reviewer: ```suggestion
It works by checking the user-supplied IPv4 or IPv6 addresses across all Objects in Cisco ASA. If a match is found, it adds the object to the group.
```
